apps/storefront/pages/api/manifest.ts

```diff
@@ -0,0 +1,17 @@
+import { createManifestHandler } from "@saleor/app-sdk/handlers/next";
+import { AppManifest } from "@saleor/app-sdk/types";
+
+export default createManifestHandler({
+  manifestFactory(context: { appBaseUrl: string }): AppManifest {
+    return {
+      id: "saleor.storefront.app",
+      version: "1.0.0", // is there any official versioning?
```

Comment:
@mmiszy ideas?

Reply:
I think we should introduce some incremental versioning one day. Doesnt have to be semver, but I assume we will end up with:

1. Apps with breaking changes, eg. "Checkout v4 works with Saleor >=4.x"
2. Clear versioning for semi-technical people, where its easy to compare that version 3.1 is newer than 3.0

Commits are good as build ids for Sentry but not so good for end users